scripts/generate-og-images.js

```
@@ -0,0 +1,201 @@
+  const stats = fs.statSync(outputPath)
+  console.log(
+    `✓ Generated ${path.relative(process.cwd(), outputPath)} (${Math.round(stats.size / 1024)}KB)`
+  )
+}
+
+Promise.allSettled(LOCALES.map(locale => generateBlogOG(locale))).then(
+  results => {
+    const failures = results.filter(r => r.status === 'rejected')
+    if (failures.length > 0) {
+      failures.forEach(f =>
+        console.error('OG image generation failed:', f.reason)
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)